sdk/eventhub/azure-eventhubs/azure/eventhub/_consumer.py

```diff
@@ -139,15 +142,38 @@ def _message_received(self, message):
         self._last_received_event = event_data
         self._on_event_received(event_data)
 
+    def _open(self):
+        """Open the EventHubConsumer/EventHubProducer using the supplied connection.
+
+        """
+        # pylint: disable=protected-access
+        if not self.running:
+            if self._handler:
+                self._handler.close()
+            auth = self._client._create_auth()
+            self._create_handler(auth)
+            self._handler.open(
+                connection=self._client._conn_manager.get_connection(self._client._address.hostname, auth)  # pylint: disable=protected-access
+            )
+            self.handler_ready = False
+            self.running = True
+
+        if not self.handler_ready:
+            if self._handler.client_ready():
+                self._max_message_size_on_link = self._handler.message_handler._link.peer_max_message_size \
+                                                 or constants.MAX_MESSAGE_LENGTH_BYTES  # pylint: disable=protected-access
```

Comment:
I don't think we need the max_message_size stuff - this is not used by consumer and is handled at the client level now anyway.

Reply:
Yes, removed it